libraries/SITL/SIM_Airspeed_DLVR.h

```diff
@@ -2,6 +2,14 @@
 
 #include "SIM_I2CDevice.h"
 
+#include <AP_HAL/AP_HAL_Boards.h>
+
+#ifndef AP_SIM_AIRSPEEDDLVR_ENABLED
```

Comment:
These should go in `SIM_config.h`

Also, please add underscores in each of them:

```suggestion
#ifndef AP_SIM_AIRSPEED_DLVR_ENABLED
```

Reply:
Thanks for the review. Updated in 92b8e86eb1546be8c7ff28839fe0905c642b802d